code/modules/mining/abandoned_crates.dm

```diff
@@ -11,6 +11,17 @@
 	var/codelen = 4
 	tamperproof = 90
 
+/obj/structure/closet/crate/proc/pathorlist_to_loot(var/loot)//needs this path for necro crates
```

Comment:
outdated comment

Reply:
Mentioning necrocrates is cause they share that proc, I just didn't know if I should stick the proc in crates.dm or leave it near a more relevant code.

Reply:
throw it in crates.dm and have the comment say something like `// used in abandoned crates and necro chests`